packages/esm-sms-app/src/providers-dashboard.scss

```diff
@@ -0,0 +1,75 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@use '@carbon/colors';
+@import '@openmrs/esm-styleguide/src/vars';
+
+.mainWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  gap: spacing.$spacing-07;
+  padding: spacing.$spacing-09 10rem;
+
+  &.overlayOpen {
+    padding-right: 1rem;
+  }
+}
+
+.tabsContainer {
+  :global(.cds--tab-content) {
+    padding: 0
+  }
+}
+
+.tabList {
+  :global(.cds--tabs__nav-item--selected) {
+    background-color: colors.$white-0;
+    box-shadow: inset 0 2px 0 0 var(--brand-03) !important;
+    border: none !important
+  }
+
+  :global(.cds--tabs__nav-link) {
+    border: none !important;
+  }
+}
+
+.tabPanel {
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem;
+  padding: 3rem 1rem 1rem;
```

Comment:
Wherever possible, please use Carbon's layout tokens / spacing tokens to specify distances. In places where this is not possible, please add a comment explaining the value chosen and why (e.g. below in the calc using 26.25rem).

Reply:
fixed, comments added